content/en/docs/install-setup/installing/create-cluster/eks.md

```diff
@@ -60,7 +60,7 @@ module "eks-jx" {
 ```
 
 The IAM user does not need any permissions attached to it.
-For more information, refer to [Configuring Vault for EKS](https://jenkins-x.io/docs/getting-started/setup/boot/clouds/amazon/#configuring-vault-for-eks) in the Jenkins X documentation.
+For more information, refer to [Configuring Vault for EKS](https://jenkins-x.io/docs/install-setup/installing/boot/clouds/amazon/#configuring-vault-for-eks) in the Jenkins X documentation.
```

Comment:
Do you mind making this a site relative link, aka '/docs/install-setup/installing/boot/clouds/amazon/#configuring-vault-for-eks'? This way the wrong link gets captured by the link checker (in case it moves again) and the content stays independent of the hosting domain.